packages/integrations/src/openmediavault/openmediavault-integration.ts

```diff
@@ -66,7 +66,7 @@ export class OpenMediaVaultIntegration extends Integration {
 
     return {
       version: systemResult.data.response.version,
-      cpuModelName: systemResult.data.response.cpuModelName,
+      cpuModelName: systemResult.data.response.cpuModelName && "Unknown CPU",
```

Comment:
Is the goal here to show the text `Unknown CPU`? If so this will not work and should be replaced with the following:
```suggestion
      cpuModelName: systemResult.data.response.cpuModelName ?? "Unknown CPU",
```